suitcase/spec.py

```diff
@@ -1008,19 +1010,13 @@ def _get_acq_time(start, default_value=-1):
 
 def _get_plan_name(start):
     plan_name = start['plan_name']
-    if plan_name not in _BLUESKY_PLAN_NAMES:
-        warnings.warn(
```

Comment:
I think this part makes sense to keep; at least, I don't see why we'd want to change it as part of this PR updating suitcase to work with the new databroker API.

Reply:
OK. Let's still return Others, if there is no match.

Reply:
I'm ok dropping the warning (and talked about this with Li) in person.  The fix for the warning would be to teach this code about more blusky name -> spec name mapping which is not really something we want to do.

The motivation to switching back to 'Other' is not knowing if spec readers have a white-listed list of names and will choke if we provide names outside of that.